webview/js/api.py

```diff
@@ -5,21 +5,28 @@
     api: {},
 
     _createApi: function(funcList) {
-        for (var i = 0; i < funcList.length; i++) {
-            var funcName = funcList[i].func;
-            var params = funcList[i].params;
+        funcList.forEach(({ func: funcName, params }) => {
```

Comment:
Even though MSHTML is deprecated, it would be nice to preserve minimal compatibility. So please no arrow functions or string literals.

Reply:
Understandable. The newest commit fixes this.